client/ayon_core/hosts/houdini/plugins/create/create_model.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+"""Creator plugin for creating Model product type.
+
+Note:
+    Currently, This creator plugin is the same as 'create_pointcache.py'
```

Comment:
I think something's wrong on the design if exposing a new type that's basically the same as another requires duplicating all this code...

Reply:
I had this thought of product aliases. _I've no idea how to use product aliases in code._
because there's no difference between 'create_pointcache.py' and `create_model.py` other than they have different `product_type`. 

Also, I want to point out that `create_model.py` in this PR is just a first draft. once publishing `multiple representations` is supported in Houdini addon. this creator plugin will be updated!